lib/features/mailbox/presentation/mailbox_view_web.dart

```diff
@@ -159,6 +161,12 @@ class MailboxView extends GetWidget<MailboxController> with AppLoaderMixin, Popu
             return _buildUserInformation(context);
           }),
           _buildLoadingView(),
+          AppConfig.appGridDashboardAvailable && _responsiveUtils.isWebNotDesktop(context)
+            ? _buildAppGridDashboard(context)
+            : const SizedBox.shrink(),
+          const SizedBox(height: 8),
+          const Divider(color: AppColor.colorDividerMailbox, height: 0.5, thickness: 0.2),
```

Comment:
You should hide it for it if `AppGridDashboard` does not support.

Reply:
After fixed: 

<img width="314" alt="image" src="https://user-images.githubusercontent.com/6462404/196882569-e3c411c1-b596-4bef-b385-be20b8430628.png">